internal/pkg/config/config.go

```diff
@@ -29,15 +29,40 @@ import (
 	"github.com/google/go-flow-levee/internal/pkg/config/regexp"
 )
 
-// FlagSet should be used by analyzers to reuse -config flag.
-var FlagSet flag.FlagSet
-var configFile string
+var (
+	// FlagSet should be used by analyzers to reuse -config flag.
+	FlagSet                    flag.FlagSet
+	configFile                 string
+	validFuncMatcherFields     map[string]bool
```

Comment:
I think it would be preferable to move each of these close to the struct that contains the actual field definitions. It would make it easier to verify that the set of strings is correct.

Reply:
I prefer to have global variables on top, so we don't miss initiating them...